tests/cylc-poll/16-execution-time-limit.t

```diff
@@ -72,6 +73,7 @@ ACTUAL_POLL_TIME=$(sed -n \
 # Test execution timeout polling.
 # Main loop is roughly 1 second, but integer rounding may give an apparent 2
 # seconds delay, so set threshold as 2 seconds.
+sleep 1
```

Comment:
Feeling sleepy?

Reply:
Made test pass reliably. Please can you double check that this is ok in terms of f'nality?

Reply:
I can't see how these sleeps would improve reliability. The next command works with the variables computed on the previous lines.

Reply:
I think for this PR we can leave just the changes for `isodatetime` => `metomi.isodatetime`, and move the work around reliability for this test to #3224 or #2894 .